dockerfiles/dev

```diff
@@ -0,0 +1,7 @@
+FROM python:3.13.7-slim
+LABEL org.opencontainers.image.source=https://github.com/con/vandermeerlab-to-bids
+LABEL org.opencontainers.image.description="A Docker image of the main developer branch for the nwb2bids project."
+
+ADD ./ nwb2bids
+RUN cd nwb2bids && pip install .[all]
+CMD ["python -m"]
```

Comment:
what actually gets executed here? I would have expected module name.

Nitpick: IIUC it should be ["python", "-m]

Reply:
You call `nwb2bids` on the Docker container once it is running, just as if it were a locally installed CLI

Based off the NeuroConv instructions: see part 6 of https://neuroconv.readthedocs.io/en/stable/user_guide/docker_demo.html#neuroconv-docker-demo

Reply:
> Nitpick: IIUC it should be ["python", "-m]

Indeed `CMD` should usually be exec-formatted rather than shell. See this nice discussion: https://stackoverflow.com/a/72444233

I worry my current usage might depend on shell only however. We will know more once I add documentation and tests for the Docker usage (follow-up PR)

At that point I will try to change it and see if it breaks anything